docs/source/contributing/developing.adoc

```diff
@@ -55,7 +58,10 @@ $ export GOPATH=$HOME/work
 $ export PATH=$PATH:$GOPATH/bin
 ----
 
-NOTE: On Windows operating systems, you use the UI or use `setx` to set the environment variables.
```

Comment:
'Setx'? This depends on using Command Prompt or PowerShell... But 'setx', never used or heard of.

Reply:
I believe `setx` was recommended here due to it being similar to setting Windows environment variables through the GUI. `set` will temporarily assign a value to an environment variable in the current shell, whereas `setx` will set the value globally (but not immediately -- the shell must be restarted).

I agree that we should mention PowerShell's usage of `$env` and `[Environment]::SetEnvironmentVariable` in addition to Command Prompt's `set` and `setx`.

I don't want to go into too much detail here regarding setting environment variables, though -- this seems like an appropriate place to mention the commands used, then link to additional information for those who need it.

Reply:
> But 'setx', never used or heard of.

I heard it :smile: However, for Powershell, it might be different https://stackoverflow.com/a/1333717/1120530.

`setx` way mentioned here https://stackoverflow.com/a/8788828/1120530 didn't work for me.

Reply:
@robin-owen Any comment on it?

Reply:
@budhram Yes, agreed, we could use a simple explanation here. I'm in favor of simply mentioning that you can use the UI or shell-specific commands with links to additional information for PowerShell and Command Prompt.

Reply:
@robin-owen Similar to this https://github.com/minishift/minishift/pull/1684#discussion_r150750447

Reply:
@budhrg Yes, I was replying to both comment threads at once. :)

Reply:
@gbraad @budhrg Please see the updated note with links to detailed information for Command Prompt and PowerShell.

Does this work for you?

Reply:
Works for me.